events/beam-robotics.html

```diff
@@ -0,0 +1,154 @@
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-book"></span> Student Guide</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-chalkboard-teacher"></span> Instructor Guide</a>
+      </div>
+    </div>
+  </div>
+
+  <div class="event-section event-section-reverse">
+    <div class="grid-col">
+      <img class="img-responsive" src="{{site.baseurl}}/img/beam-robotics/beetle.svg" alt="a beetle and leaf from a game you'll code in our week 2 lesson"/>
+    </div>
+    <div>
+      <h2 class="title text-1.5x">
+        <a class="tla-link" href="#" target="_blank" rel="noopener noreferrer">Week 2: Robot Coding Basics</a> 
+      </h2>
+      <p>
+        Explore more with mBlock and learn how to make a bug move just like how a robot would. By the end of this lesson we will have completed <b><a class="tla-link" href="https://planet.mblock.cc/project/445974" target="_blank" rel= "noopener noreferrer" >our second game</a></b>! 
+      </p>
+      <div class="button-container">
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fab fa-youtube"></span> Video</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-book"></span> Student Guide</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-chalkboard-teacher"></span> Instructor Guide</a>
+      </div>
+    </div>
+  </div>
+  
+  <div class="event-section">
+    <div class="grid-col">
+      <img class="img-responsive" src="{{site.baseurl}}/img/beam-robotics/m_bot.svg" alt="a blue mbot with wheels"/>
+    </div>
+    <div>
+      <h2 class="title text-1.5x">
+        <a class="tla-link" href="#" target="_blank" rel="noopener noreferrer">Week 3: Assembling our Robot</a>  
+      </h2>
+      <p>
+        Beep boop beep boop. Learn how to put together your robot! Once we’ve assembled our robot, <a class="tla-link" href="https://www.asmebruins.com/" target="_blank" rel="noopener noreferrer"> ASME at UCLA</a> will come in and teach you a bit about the fancy robots and Battle Bots we make in university. 
+    </p>
+      <div class="button-container">
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fab fa-youtube"></span> Video</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-book"></span> Student Guide</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-chalkboard-teacher"></span> Instructor Guide</a>
+      </div>
```

Comment:
since we don't have these available yet, can we leave these out? ill bop you when we have the info!